people_velocity_tracker/scripts/tracker.py

```diff
@@ -91,9 +91,14 @@ def __init__(self):
         self.people = {}
         self.TIMEOUT = rospy.Duration(rospy.get_param('~timeout', 1.0))
         self.sub = rospy.Subscriber('/people_tracker_measurements',
-                                    PositionMeasurementArray, self.pm_cb)
-        self.mpub = rospy.Publisher('/visualization_marker', Marker)
-        self.ppub = rospy.Publisher('/people', People)
+                                    PositionMeasurementArray,
+                                    self.pm_cb)
+        self.mpub = rospy.Publisher('/visualization_marker',
+                                    Marker,
+                                    queue_size=0)
```

Comment:
queue_size = 0 is generally a bad idea -- see http://wiki.ros.org/rospy/Overview/Publishers%20and%20Subscribers#Choosing_a_good_queue_size

I would suggest something like 10 for this message